test/testutil.py

```diff
@@ -147,3 +149,270 @@ def create_filesystem_customizations(rootfs: str):
     "-v", "/var/lib/containers/storage:/var/lib/containers/storage",
     "--security-opt", "label=type:unconfined_t",
 ]
+
+
+def get_ip_from_default_route():
+    default_route = subprocess.run([
+        "ip",
+        "route",
+        "list",
+        "default"
+    ], check=True, capture_output=True).stdout
+    return default_route.split()[8].decode("utf-8")
+
+
+@dataclasses.dataclass
+class GPGConf:
+    _key_params_tmpl: str = """
+      %no-protection
+      Key-Type: RSA
+      Key-Length: {key_length}
+      Key-Usage: sign
+      Name-Real: Bootc Image Builder Tests
+      Name-Email: {email}
+      Expire-Date: 0
+    """
+    _base_dir: str = "/tmp/bib-tests"
+    _home_dir: str = f"{_base_dir}/.gnupg"
+    _pub_key_file: str = f"{_base_dir}/GPG-KEY-bib-tests"
+    _email: str = "bib-tests@redhat.com"
+    _key_length: str = "3072"
+    _key_params: str = _key_params_tmpl.format(key_length=_key_length, email=_email)
+
+    @property
+    def base_dir(self) -> str:
+        return self._base_dir
+
+    @base_dir.setter
+    def base_dir(self, base_dir: str) -> None:
+        self._base_dir = base_dir
+        self._home_dir = f"{base_dir}/.gnupg"
+        self._pub_key_file = f"{base_dir}/GPG-KEY-bib-tests"
+
+    @property
+    def home_dir(self) -> str:
```

Comment:
Sorry, two maybe silly questions:
1. should we be smart about the base dir here too, i.e. just like above build the home dir relative to the base dir?
2. if not, would it make sense to just avoid the getter/setter here and use the dataclass directly (i.e. change _home_dir to home_dir) because we don't do anything with the `home_dir` in the getter/setter (unlike the base_dir above)

Reply:
Yeah, you are right, we can use the getter/setter from the dataclass so the `home_dir` can be set independently from `base_dir